src/atoms/Checkbox/Checkbox.native.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableOpacity, View } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import CheckBoxIcon from './CheckboxIcon';
+
+const CheckboxContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+`;
+
+const RippleView = styled(View)(
+  (props) => `
+  width: ${props.width}px;
+  height: ${props.height}px;
+  border-radius: ${props.width / 2}px;
+  background-color: ${props.underlayColor ? props.underlayColor : 'transparent'};
+`,
+);
+
+const TextContainer = styled.View`
+  margin-left: 4px;
+`;
+
+const Title = styled(Text)(
```

Comment:
Why is this needed? Why don't we use `<Text ... />` directly and pass props. It supports `color` and `size` both.